package.json

```diff
@@ -7,6 +7,11 @@
   "author": "Bryan Helmig <bryan@zapier.com>",
   "license": "UNLICENSED",
   "main": "zapier.js",
+  "files": [
+      "lib/*.js",
```

Comment:
I think this can be simplified to include the whole `lib` directory, maybe excluding bin (`"!lib/bin/`)

Reply:
I feel whitelisting is more straightforward and I'm not sure if we can do exclusion in `package.json`'s `files`. Can't find it in the npm's [doc](https://docs.npmjs.com/files/package.json#files).

Reply:
npm uses [Glob](https://github.com/isaacs/node-glob#glob-primer), so we can use anything that's valid in that. I don't hate the simplicity though, so it's your call